### PR TITLE
refactor: Removed deprecated values `MaterialApp.router` in v3.10.0

### PR DESCRIPTION
**Pull request by i-asimkhan**
## Description

Removed deprecated values `MaterialApp.router` in v3.10.0

## Type of Change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Code refactor
- [ ] Build configuration change
- [ ] Documentation
- [ ] Chore

### CODE REVIEW

File: packages/mirai/lib/src/framework/mirai_app.dart [hunk 1]
```diff
@@ -185,7 +185,6 @@ class MiraiApp extends StatelessWidget {
       restorationScopeId: restorationScopeId,
       scrollBehavior: scrollBehavior,
       debugShowMaterialGrid: debugShowMaterialGrid,
-      useInheritedMediaQuery: useInheritedMediaQuery,
```
Comment (divyanshub024) [inline packages/mirai/lib/src/framework/mirai_app.dart:188]: If this is deprecated, then what's the replacement for this?

File: packages/mirai/lib/src/framework/mirai_app.dart [hunk 2]
```diff
@@ -223,7 +222,6 @@ class MiraiApp extends StatelessWidget {
       actions: actions,
       restorationScopeId: restorationScopeId,
       scrollBehavior: scrollBehavior,
-      useInheritedMediaQuery: useInheritedMediaQuery,
```
Comment (divyanshub024) [inline packages/mirai/lib/src/framework/mirai_app.dart:226]: Same goes here

File: packages/mirai/lib/src/parsers/mirai_alert_dialog/mirai_alert_dialog.dart [hunk 3]
```diff
@@ -32,8 +32,10 @@ class MiraiAlertDialog with _$MiraiAlertDialog {
     String? semanticLabel,
     @Default(MiraiEdgeInsets(left: 40, right: 40, top: 24, bottom: 24))
         MiraiEdgeInsets insetPadding,
-    @Default(Clip.none) Clip clipBehavior,
-    @Default(false) bool scrollable,
+    @Default(Clip.none)
+        Clip clipBehavior,
+    @Default(false)
+        bool scrollable,
```
Comment (divyanshub024) [inline packages/mirai/lib/src/parsers/mirai_alert_dialog/mirai_alert_dialog.dart:38]: What's the reason for these formatting changes?? Is it the part of flutter upgrade?

**Review by divyanshub024 — CHANGES_REQUESTED** [review]

File: packages/mirai/lib/src/framework/mirai_app.dart (hunk 1 above)
Comment (i-asimkhan) [reply packages/mirai/lib/src/framework/mirai_app.dart:188]: As the docs say, now the inherited media query is handled by the view itself. 
```
 'Remove this parameter as it is now ignored. '
      'MaterialApp never introduces its own MediaQuery; the View widget takes care of that. '
      'This feature was deprecated after v3.7.0-29.0.pre.'
```

File: packages/mirai/lib/src/parsers/mirai_alert_dialog/mirai_alert_dialog.dart (hunk 3 above)
Comment (i-asimkhan) [reply packages/mirai/lib/src/parsers/mirai_alert_dialog/mirai_alert_dialog.dart:38]: Yeah, you are right. It has changed as per the release notes 
`Reintroduce the ability to override the coverage formatter by @liamappelbe in https://github.com/flutter/flutter/pull/100320`

File: packages/mirai/lib/src/framework/mirai_app.dart (hunk 1 above)
Comment (divyanshub024) [reply packages/mirai/lib/src/framework/mirai_app.dart:188]: Got it.

File: packages/mirai/lib/src/framework/mirai_app.dart (hunk 2 above)
Comment (i-asimkhan) [reply packages/mirai/lib/src/framework/mirai_app.dart:226]: Resolving this as same goes here too,

**Pull request merged**